viewing_party/party.py

```diff
@@ -1,23 +1,176 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_info = [title, genre, rating]
+    if None in movie_info:
+        movie = None
+    else:
+        movie = {"title": title, "genre": genre, "rating": rating}
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
 
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data,movie_title):
+    for i in range(len(user_data["watchlist"])):
+        for info in user_data["watchlist"][i].values():
+            if info == movie_title:
+                # Remove movie from watchlist
+                watched_movie = user_data["watchlist"].pop(i)
+                add_to_watched(user_data, watched_movie)

```

Comment:
This works just fine, but there's a lot going on! And I think we can make the for loop a bit more readable. By using for `movie in user_data["watchlist"]` is much more descriptive and shorter. 

Since we know the key we want to look into already, let's skip the second loop completely. 

We could also use a `break` at the end so that we aren't running the for loop more time than necessary

```suggestion
    for movie in user_data["watchlist"]:
        watched_movie = user_data["watchlist"].remove(movie)
        add_to_watched(user_data, watched_movie)
```

I like that you reused `add_to_watched` ! Nice job!